### PR TITLE
Added Recognition Page Link

### LINKED ISSUE #1520: Add Recognition Program Link to Community section
#### Current State of the Meshery.io site
image
image

#### Desired State of the Meshery.io site
Add [Recognition Program](https://badges.layer5.io/) link to the header and footer of the site, under the Community section

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Raj100679**
**Description**
I have added recognition Page link to footer and in community dropdown

This PR fixes #1520 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Raj100679** [comment]
@iArchitSharma done!

File: _data/navigation.yml [hunk 1]
```diff
@@ -37,6 +37,10 @@
       new_window: false
       external: false
       img_src: /assets/images/nav-icons/Mail.svg
+    - name: Recognition
+      link: https://badges.layer5.io/
+      new_window: true
```
Comment (leecalcote) [inline _data/navigation.yml:42]: Please add external: true

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: _data/navigation.yml (hunk 1 above)
Comment (Raj100679) [reply _data/navigation.yml:42]: Actually alignment doesn’t looks good, I can do it

Comment (iArchitSharma) [reply _data/navigation.yml:42]: @Raj100679 do it for now

Comment (Raj100679) [reply _data/navigation.yml:42]: @iArchitSharma will do it in some time

Comment (iArchitSharma) [reply _data/navigation.yml:42]: Looks good to me
image

**Pull request closed without merging**